clients/packages/polarkit/src/api/client/services/AccountsService.ts

```diff
@@ -14,11 +14,11 @@ export class AccountsService {
   constructor(public readonly httpRequest: BaseHttpRequest) {}
 
   /**
-   * Get Account
+   * Get Accounts
    * @returns AccountRead Successful Response
    * @throws ApiError
    */
-  public getAccount({
+  public getAccounts({
```

Comment:
@frankie567 Did you find and run `cd clients/packages/polarkit && pnpm generate` yourself to generate these changes or did you do them by hand? 

Just realized it's something we haven't documented in our `clients/README.md`. I feel awful if you had to change these by hand 😬

Reply:
Nah, don't worry 😄 I suspected this was automatically generated code, so I looked for the command!

Reply:
Phew :-)